django_u2f/templates/u2f/add_key.html

```diff
@@ -14,7 +13,7 @@
 </form>
 
 <script type="application/json" id="django_u2f_registration">
-  {{ registration_request|json }}
+  {{ registration_request|json_script:"registration_request" }}
```

Comment:
doesn't json_script add its own script tag?

Reply:
Yes, it does, thanks for catching that. I just removed to excess script tags.